charts/tidb-operator/templates/admission-controller-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+{{- if .Values.admissionController.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: admission-controller-certs
+data:
+  cert.pem: {{ .Values.admissionController.cert }}
+  key.pem: {{ .Values.admissionController.key }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: admission-controller-svc
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: admission-controller
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+spec:
+  ports:
+  - port: 443
+    targetPort: 443
+  selector:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: admission-controller
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tidb-admission-controller-deployment
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: admission-controller
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+spec:
+  replicas: {{ .Values.admissionController.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "chart.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: admission-controller
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "chart.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: admission-controller
+    spec:
+      serviceAccountName: admission-controller-sa
+      containers:
+        - name: tidb-admission-controller-pod
+          image: {{ .Values.operatorImage }}
+          imagePullPolicy: {{ .Values.imagePullPolicy | default "IfNotPresent" }}
+          command:
+            - /usr/local/bin/tidb-admission-controller
+            - -tlsCertFile=/etc/webhook/certs/cert.pem
+            - -tlsKeyFile=/etc/webhook/certs/key.pem
+            - -alsologtostderr
+            - -v={{ .Values.admissionController.logLevel }}
+            - 2>&1
```

Comment:
Is this necessary?

Reply:
you mean the args？

Reply:
It's necessary if the program did not init the log, and I change the method. done.

Reply:
So it's unnecessary now?

Reply:
It's not necessary now , I have remove it in yaml file.